river/base/drift_detector.py

```diff
@@ -2,16 +2,12 @@
 import numbers
 import typing
 
-from . import estimator
+from . import base
 
 
-class DriftDetector(estimator.Estimator):
+class DriftDetector(base.Base):
     """A drift detector."""
 
-    @property
-    def _supervised(self):
-        return False
-
     def __init__(self):
         super().__init__()
```

Comment:
No need for this I think.

Reply:
I removed it as the property is defined in the `Estimator` class. I am not completely sure that it applies to drift detectors.